File: solutions/javascript/RemoveDuplicatesfromSortedArray/RemoveDuplicatesfromSortedArray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let n = 0;
    for (let i = 1; i < nums.length; i++) {
        if (nums[i] != nums[n]) {
            n++;
            nums[n] = nums[i];
        }
    }
    return n+1;
};

File: solutions/java/RemoveDuplicatesfromSortedArray/RemoveDuplicatesfromSortedArray.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n = 0;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != nums[n]) {
                n++;
                nums[n] = nums[i];
            }
        }
        return n + 1;
    }
}